cmd/autoscaler/main.go

```diff
@@ -204,7 +204,8 @@ func uniScalerFactoryFunc(endpointsInformer corev1informers.EndpointsInformer, m
 
 func statsScraperFactoryFunc(endpointsLister corev1listers.EndpointsLister) func(metric *autoscaler.Metric) (autoscaler.StatsScraper, error) {
 	return func(metric *autoscaler.Metric) (autoscaler.StatsScraper, error) {
-		return autoscaler.NewServiceScraper(metric, endpointsLister)
+		podCounter := resources.NewScopedEndpointsCounter(endpointsLister, metric.Namespace, metric.Name)
```

Comment:
just inline?

Reply:
I considered it, because it's not a variable with any other purpose. 

Inline pushes it out past 120 columns. I could split the `NewServiceScraper` over two lines, but I feel that's slightly less readable.

Reply:
```
return autoscaler.NewServiceScraper(
   metric,
   resources.NewScopedEndpointsCounter(endpointsLister, metric.Namespace, metric.Name))
```
is 3 lines :) but I actually like this better. 
In any case, I am not bent on this :)

Reply:
OK, I'll keep it as-is. Fewer pushes -> less chance of integration flakes.